packages/arb-token-bridge-ui/src/util/AnalyticsUtils.ts

```diff
@@ -100,6 +104,13 @@ const eventToEventId: {
   'Fast Bridge Click: Connext': 'KFF7GMET',
   'Fast Bridge Click: Across': 'EZDV8TMY',
   //
+  'FRAX: Fast Bridge Click: Celer': '6PZJPSBO',
+  'FRAX: Fast Bridge Click: Hop': '',
+  'FRAX: Fast Bridge Click: Connext': '',
+  'FRAX: Fast Bridge Click: Across': '',
```

Comment:
@spsjvc thinking of a better way to cover these cases so that typescript doesn't complain if they are missing. If you have any ideas to throw at me feel free